CommonCompilerConfig.cmake

```diff
@@ -281,7 +281,7 @@ else()
 
 	# Hide all symbols in DLLs except the ones with explicit visibility;
         # see https://gcc.gnu.org/wiki/Visibility
-        set(cxx_flags_unoptimized ${cxx_flags_unoptimized} -fvisibility-inlines-hidden -fvisibility=hidden)
+        set(cxx_flags_unoptimized ${cxx_flags_unoptimized} -fvisibility-inlines-hidden )
```

Comment:
is it bad thing we remove the hiddent visibility? Ideally keep it for static builds